Sources/PostgresNIO/New/PostgresBackendMessage.swift

```
@@ -131,9 +135,9 @@ extension PostgresBackendMessage {
             
         case .rowDescription:
             return try .rowDescription(.decode(from: &buffer))
-            
-        case .copyData, .copyDone, .copyInResponse, .copyOutResponse, .copyBothResponse, .functionCallResponse, .negotiateProtocolVersion:
-            preconditionFailure()
+
+        case .copyData, .copyDone, .copyOutResponse, .copyBothResponse, .functionCallResponse, .negotiateProtocolVersion:
+            preconditionFailure("Unknown message kind: \(messageID)")
```

Now that we have touched that line, lets please throw an error here.